assets/js/components/Dashboard/NewPaMessagePage.tsx

```diff
@@ -130,13 +149,27 @@ const NewPaMessagePage = () => {
                   End
                 </Form.Label>
                 <div className="datetime-picker-group">
-                  <DatePicker
-                    selectedDate={endDate}
-                    onChange={setEndDate}
-                    minDateString={startDate}
+                  <Form.Control
+                    className="date-picker picker"
+                    type="date"
                     id="end-date-picker"
+                    name="end-date-picker-input"
+                    value={endDate}
+                    onChange={(event) => setEndDate(event.target.value)}
+                  />
+                  <Form.Control
+                    type="time"
+                    className="time-picker picker"
+                    value={endTime}
+                    onChange={(event) => setEndTime(event.target.value)}
                   />
-                  <TimePicker selectedTime={endTime} onChange={setEndTime} />
+                  <Button
+                    className="service-time-link"
+                    variant="link"
+                    onClick={() => setEndTime("02:59")}
```

Comment:
I think this should also be `03:00`, per [this thread](https://mbta.slack.com/archives/C06211Q5YAZ/p1721932270002629) and the updated [Start/End of Service](https://www.notion.so/mbta-downtown-crossing/Start-of-Service-End-of-Service-change-dcf3f99e296e4f02aeaeab362b495119) guidance.

Reply:
Ahh, thanks for the link!